templates/vue/src/components/modals/NodeModal/index.vue

```diff
@@ -930,18 +914,35 @@ export default {
         if (!validPreviousAnswers) {
           errMsgs.push("Please select a previous activity to display")
         }
+        if (this.node.typeData.activity.questions[0].answerTypes.list.enabled) {
+          let listQuestion = this.node.typeData.activity.questions[0].answerTypes
+            .list
+          let minValue = parseInt(listQuestion.minFields, 10)
+          if (minValue < 1) {
+            errMsgs.push("Minimum # of fields for list activity must be >= 1")
```

Comment:
It would be more clear to type the word "number" in fully rather than using the "#" shorthand (here and below / elsewhere)